### PR TITLE
Added telemetry subscriptions.

### PR DESCRIPTION
**Pull request by irsdkv**
Added:
 
- AttitudeAngularVelocityBody: current angular speed from [ATTITUDE_QUATERNION](https://mavlink.io/en/messages/common.html#ATTITUDE_QUATERNION) message.

-   ActuatorControlTarget: current vehicle [attitude and body angular rates](https://mavlink.io/en/messages/common.html#SET_ACTUATOR_CONTROL_TARGET).
    
-  ActuatorOutputStatus: current raw values of the [actuator outputs](https://mavlink.io/en/messages/common.html#ACTUATOR_OUTPUT_STATUS).

Related PR in proto: https://github.com/mavlink/MAVSDK-Proto/pull/97

### CODE REVIEW

**Comment by julianoes** [comment]
Thanks! I'll review it but it might not happen before Monday.

File: src/backend/test/telemetry_service_impl_test.cpp [hunk 1]
```diff
@@ -125,6 +137,20 @@ class TelemetryServiceImplTest : public ::testing::Test {
...
+        const std::vector<ActuatorControlTarget>& actuator_control_target_events) const;
+    ActuatorControlTarget
+    createActuatorControlTarget(const uint8_t group, const float* controls) const;
+    std::future<void> subscribeActuatorControlTargetAsync(
+        std::vector<ActuatorControlTarget>& actuator_control_target_events) const;
+
+    void checkSendsActuatorOutputStatusEvents(
+        const std::vector<ActuatorOutputStatus>& actuator_output_status_events) const;
+    ActuatorOutputStatus
+    createActuatorOutputStatus(const uint8_t group, const float* controls) const;
```
Comment (JonasVautherin) [inline src/backend/test/telemetry_service_impl_test.cpp]: Does it have to be pointer? What about e.g. a `const vector&`?

Comment (irsdkv) [reply src/backend/test/telemetry_service_impl_test.cpp]: Agree. Will do this

Comment (irsdkv) [reply src/backend/test/telemetry_service_impl_test.cpp]: Changed createActuatorControlTarget in the same way.

**Comment by JonasVautherin** [comment]
Let's now update the `proto` submodule and sync with `develop`, and I think we can merge :+1:

**Comment by julianoes** [comment]
```
../../../../../src/plugins/telemetry/telemetry_impl.cpp: In member function 'void mavsdk::TelemetryImpl::process_actuator_control_target(const mavlink_message_t&)':

../../../../../src/plugins/telemetry/telemetry_impl.cpp:732:68: error: taking address of packed member of '__mavlink_actuator_control_target_t' may result in an unaligned pointer value [-Werror=address-of-packed-member]

  732 |         actuator_control_target.group_mlx, actuator_control_target.controls);

      |                                            ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~

../../../../../src/plugins/telemetry/telemetry_impl.cpp: In member function 'void mavsdk::TelemetryImpl::process_actuator_output_status(const mavlink_message_t&)':

../../../../../src/plugins/telemetry/telemetry_impl.cpp:746:86: error: taking address of packed member of '__mavlink_actuator_output_status_t' may result in an unaligned pointer value [-Werror=address-of-packed-member]

  746 |     set_actuator_output_status(actuator_output_status.active, actuator_output_status.actuator);

      |                                                               ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
```

:thinking:

**Comment by irsdkv** [comment]
Im also think about that. But at this moment I suppose that this ok, because of `set_actuator_output_status` is private method.
Only alternate way is using vectors, but I think that this is redundant work for the CPU (sorry, I'm embedded developer :smile: )

**Comment by irsdkv** [comment]
Also may be `std::array` can be used

**Comment by julianoes** [comment]
> Also `std::array` can be used

Sounds good.

**Comment by julianoes** [comment]
```
Generating code for file plugin_bas/home/user/DronecodeSDK/tools/docs/install/include/mavsdk/plugins/telemetry/telemetry.h:568: warning: The following parameters of mavsdk::Telemetry::set_rate_actuator_control_target_async(double rate_hz, result_callback_t callback) are not documented:
  parameter 'callback'
/home/user/DronecodeSDK/tools/docs/install/include/mavsdk/plugins/telemetry/telemetry.h:578: warning: The following parameters of mavsdk::Telemetry::set_rate_actuator_output_status_async(double rate_hz, result_callback_t callback) are not documented:
  parameter 'callback'
```

**Comment by julianoes** [comment]
And:
```
../../../../../src/plugins/telemetry/telemetry_impl.cpp: In member function 'void mavsdk::TelemetryImpl::process_actuator_control_target(const mavlink_message_t&)':
../../../../../src/plugins/telemetry/telemetry_impl.cpp:731:91: error: taking address of packed member of '__mavlink_actuator_control_target_t' may result in an unaligned pointer value [-Werror=address-of-packed-member]
  731 |     auto controls = reinterpret_cast<const std::array<float, 8>&>(actuator_control_target.controls);
      |                                                                   ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
../../../../../src/plugins/telemetry/telemetry_impl.cpp: In member function 'void mavsdk::TelemetryImpl::process_actuator_output_status(const mavlink_message_t&)':
../../../../../src/plugins/telemetry/telemetry_impl.cpp:748:79: error: taking address of packed member of '__mavlink_actuator_output_status_t' may result in an unaligned pointer value [-Werror=address-of-packed-member]
  748 |         reinterpret_cast<const std::array<float, 32>&>(actuator_output_status.actuator);
      |                                                        ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
cc1plus: all warnings being treated as errors
```

**Comment by irsdkv** [comment]
@julianoes can you please merge PR if it possible?

File: tools/generate_from_protos.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@ set -e
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-proto_dir="${script_dir}/../proto/protos"
+proto_dir="${script_dir}/../src/backend/proto/protos"
```
Comment (julianoes) [inline tools/generate_from_protos.sh]: Is this correct?

Comment (irsdkv) [reply tools/generate_from_protos.sh]: Yes, this is fix. Apparently, protos was moved before. 
gRPC headers and SRC generated correctly now

Comment (julianoes) [reply tools/generate_from_protos.sh]: oh ok!

**Comment by julianoes** [comment]
Thanks @irsdkv!

**Pull request merged**